### DISCUSSION TITLE
By pages/xxx.vue, I want to access image or video files from the server path like "/home/my_resource", how to do it?

### DISCUSSION THREAD

**Discussion by 45592858** [nuxt/nuxt#23427] [Questions]
like following:

```
1. <template>
2.   <a-card hoverable style="width: 240px; margin: 15px; padding: 20px;">
3.      <template #cover>
4.        <img :src="/home/my_resource/xxxx.png" />
5.      </template>
6.    </a-card>
7. </template>
```

Since the resource file is on the server path, and cannot put them into public & assets ...

**Comment by 45592858** [comment]
Anyone can help me ...

**Comment by AloisSeckar** [comment]
Try setting `app.cdnURL` to `/home/my_resource`

https://nuxt.com/docs/api/configuration/nuxt-config#cdnurl

**Reply by 45592858** [reply]
When I run: NUXT_APP_CDN_URL=/home node /home/jetion-dev/www/nuxtjs/server/index.mjs

It will prompt below error: 
[Vue Router warn]: No match found for location with path "/home/_nuxt/entry.6bf5fc17.js"

How to do?
